frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx

```diff
@@ -184,7 +184,7 @@ export const SidePanelSupport = (): JSX.Element => {
 
                     <Section title="Ask the community">
                         <p>
-                            Questions about features, how to's, or use cases? There are thousands of discussions in our
+                            Questions about features, how tos, or use cases? There are thousands of discussions in our
```

Comment:
Actually I think the _correct_ way is to hyphenate it ;)

Reply:
You're correct, it's a compound noun without a possessive

```suggestion
                            Questions about features, how-tos, or use cases? There are thousands of discussions in our
```